tests/integration/src/test/scala/mesosphere/marathon/integration/TaskUnreachableIntegrationTest.scala

```diff
@@ -223,12 +224,17 @@ class TaskUnreachableIntegrationTest extends AkkaIntegrationTest with EmbeddedMa
       waitForEventMatching("Task is declared killed") {
         matchUnknownTerminatedEvent(Task.Id(taskId).instanceId)
       }
+
+      // delete the pod to never see it again
+      val newInstanceId = marathon.status(pod.id).value.instances.head.id
+      marathon.deleteInstance(pod.id, newInstanceId, wipe = true) should be(OK)
     }
 
     "wipe pod instances without persistent volumes" in {
       Given("a pod with persistent volumes")
-      val pod = simplePod("simple-pod-with-one-instance-wipe-test").copy(
-        instances = 1
+      val pod = simplePod("simple-pod-with-one-instance-wipe").copy(
+        instances = 1,
+        unreachableStrategy = UnreachableDisabled
```

Comment:
Can you add a quick note to the summary to describe why this was added? (I think I know why, but, summary just says "we add eventually"

Reply:
...and a ticket, saying that this can be removed once Instance Goal Phase 2 has landed ;)

Reply:
...and what Tim said ^^: I definitely want to see a lengthy and "newbie developer" friendly comment about why that is needed.